mlrun/projects/project.py

```diff
@@ -58,7 +58,7 @@
 import mlrun.utils.regex
 import mlrun_pipelines.common.models
 from mlrun.alerts.alert import AlertConfig
-from mlrun.common.schemas.alert import AlertTemplate
+from mlrun.common.schemas.alert import AlertActivation, AlertTemplate
```

Comment:
use import x instead of from x import y
https://github.com/mlrun/mlrun/blob/4708bc367cc80a1d8ed74d5f0a1f72f9434dbfa5/CONTRIBUTING.md#L335:~:text=When%20importing%20from%20local%20code%2C%20NEVER%20use%3A%20from%20X%20import%20Y%20instead%20use%3A%20import%20X%20or%20import%20X%20as%20Y.%20For%20external%20packages%2C%20it%27s%20acceptable%20to%20use%20from%20X%20import%20Y%20since%20they%20won%27t%20try%20to%20import%20back%20to%20our%20code.